vms/platformvm/state/merkle_state.go

```diff
@@ -388,6 +388,10 @@ type merkleState struct {
 	rewardUTXOsDB    database.Database
 }
 
+func (ms *merkleState) MerkleView() (merkledb.TrieView, error) {
+	return ms.merkleDB.NewView(context.Background(), merkledb.ViewChanges{})
```

Comment:
I think we always pass empty changes here, since currently we apply to state and commit as a single thing, so there should never be pending changes